archinstall/lib/user_interaction.py

```diff
@@ -263,7 +265,7 @@ def select_disk(dict_o_disks):
 		elif drive.isdigit():
 			drive = int(drive)
 			if drive >= len(drives):
-				raise DiskError(f'Selected option "{drive}" is out of range')
+				raise Dis1ror(f'Selected option "{drive}" is out of range')
```

Comment:
This seems like a mistake?

Reply:
@Torxed if you're fine with this, could you correct this line and merge this in? We can further clean up and refine this code to make it up to standards before 2.2.0 is released? Would like to see this feature land for 2.2.0. It looks good enough for testing at least.

Reply:
I'll get on it soon, merging in to master in ~8h :)